csharp/ql/src/Security Features/CWE-327/InsecureSQLConnection.ql

```diff
@@ -32,6 +32,11 @@ module InsecureSqlConnectionConfig implements DataFlow::ConfigSig {
         oc.getType().getName() = "SqlConnectionStringBuilder"
         or
         oc.getType().getName() = "SqlConnection"
+      ) and 
+      not exists(MemberInitializer mi | 
```

Comment:
Auto-format check fails.

Reply:
hi sorry this fell under the radar - ran codeql query format on file